### PR TITLE
EventThreadJOptionPane.java  refactor

### PR DESCRIPTION
**Pull request by DanVanAtta**
Cleaning up, no functional changes

This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/926)

### CODE REVIEW

**Comment by DanVanAtta** [comment]
@ron-murhammer , in case you are holding off on review, please do not hold off! Who knows how long we will be doing release testing, if all PRs are queued for a month, we're going to have a HUGE list, and likely are going to have a lot of bugs when we merge all that in (given that many changes without things getting any integration testing, it's a recipe for bugs and bad merge conflicts).

Note, merging anything at this point is *not* increasing the test scope. We are doing *full* regression testing at the moment, so it's actually better to merge as much as possible really. The idea being that we'll take better advantage of the full regression testing  by adding more content in for that testing.

Also of note, the serialization proxy stuff would actually be really good to have in the 1.9 release. That change adds resiliency for save game compatibility, it would allow 1.9 games to be played potentially for longer on updated engines.

**Comment by ron-murhammer** [comment]
@DanVanAtta I'm not just haven't had much time.

File: src/games/strategy/util/EventThreadJOptionPane.java [hunk 1]
```diff
@@ -38,29 +52,10 @@ private static JScrollPane createJLabelInScrollPane(final String message) {
...
     return scroll;
   }
 
-  public static void showMessageDialog(final Component parentComponent, final Object message, final String title,
-      final int messageType, final boolean useJLabel, final CountDownLatchHandler latchHandler) {
-    if (SwingUtilities.isEventDispatchThread()) {
-      JOptionPane.showMessageDialog(parentComponent, useJLabel ? createJLabelInScrollPane((String) message) : message,
-          title, messageType);
-      return;
-    }
```
Comment (RoiEXLab) [inline src/games/strategy/util/EventThreadJOptionPane.java]: Can't find this in the newer code... Is this no longer needed?

**Comment by RoiEXLab** [comment]
@DanVanAtta looks good overall. It's a bit hard to see which code was just moved, and which code was actually modified, but I should be alright.

**Pull request merged**